Tetfuza/TetfuzaBackend/TetfuzaBackend.cs

```diff
@@ -6,6 +6,24 @@
 
 namespace Tetfuza
 {
+    public class SystemConsole : IConsole
```

Comment:
technically, i should put this in a separate file, but I didn't see a big reason to do so.